src/ServiceControl.Transports.ASQ/transport.manifest

```diff
@@ -1,9 +1,9 @@
 ﻿{
-  "Version": "1.0.0",
   "Definitions": [
     {
       "Name": "AzureStorageQueue",
       "DisplayName": "Azure Storage Queues",
+      "AssemblyName": "ServiceControl.Transports.ASQ",
       "TypeName": "ServiceControl.Transports.ASQ.ASQTransportCustomization, ServiceControl.Transports.ASQ",
```

Comment:
I know we can't change this now because of backward compatibility and in general I also like having the assembly name separate. What looks awkward now is that we have the assembly name and the type name contains it too.

Reply:
I had looked into this a bit, and I believe we can change this if we want without migration / back compat concerns. However, I have currently left it in because the assembly portion is actually required to be there for the `Type.GetType` calls to be able to properly load the types.

If we wanted to remove it from the manifest, we'd then have to reassemble at runtime to pass into the `Type.GetType` calls. Not sure that's worth doing?

Reply:
I did also move where the check occurs, so that it only runs if the plugin folder actually has a copy of the assembly also. That makes it have to check the default context much less often.